doc/en/user/source/security/urlchecks.rst

```diff
@@ -0,0 +1,67 @@
+.. _security_urlchecks:
+
+URL Checks
+==========
+
+The URL checks page controls the checks that are performed on user provided URLs that
+GeoServer will use to access remote resources.
+
+Currently, the checks are performed on the following functionality:
+
+- WMS GetMap requests with remote SLD stylesheets (``sld``` parameter)
+- Remote icons referenced by styles (access to icons in the data directory is always allowed)
+- WMS GetMap in feature portrayal mode (``REMOTE_OWS`` and ``REMOTE_OWS_TYPE`` parameters)
+- WPS remote inputs, either as GET or POST requests
+
+More functionality might go under this kind of scrutiny in the future, check this page
+for eventual updates.
+
+Configuration of URL checks
+---------------------------
+
+The "URL checks" page under the Security section allows configuration of the URL checks.
+
+.. figure:: images/urlchecks.png
```

Comment:
This image url checks.png appears to be missing

Reply:
Right! Just added it back.